integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktApplication.java

```diff
@@ -25,17 +28,43 @@ public ServletRegistrationBean servletNoArk() {
 
     public static void main(String[] args) {
         try {
+            if(!validateJCE()){
+                logMissingJCE(null);
```

Comment:
Det er ikke ansett for å være god skikk i Java å sende null parametre til metoder for da må man lese koden for å forstå hvorfor null blir sendt. Da er det bedre å overlastemetodene til en som ikke tar parametre og en som tar parametre